app/deliver_grant_funding/routes.py

```diff
@@ -204,7 +204,14 @@ def list_grants() -> Response | str:
     return render_template("deliver_grant_funding/grant_list.html", grants=grants)
 
 
-@deliver_grant_funding_blueprint.route("/grant/users/<uuid:grant_id>", methods=["GET"])
+@deliver_grant_funding_blueprint.route("/grant/<uuid:grant_id>", methods=["GET"])
+@mhclg_login_required
+def view_grant(grant_id: UUID) -> str:
```

Comment:
want to make these consistent too?
```suggestion
def view_grant(grant_id: UUID) -> ResponseReturnValue:
```